packages/components/src/__future__/Tag/Tag/_docs/Tag-migration-guide.stories.tsx

```diff
@@ -24,105 +23,16 @@ const meta = {
 
 export default meta
 
-/** * This is a stand-in component for the legacy Tag's bake in LiveIcon - we should consider adding this as an actual component or replacing it */
-const LiveIconComponent = (): JSX.Element => (
-  <span className={styles.liveIcon}>
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_base}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_1}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_2}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-    <LiveIcon
-      role="presentation"
-      classNameOverride={styles.liveIcon_3}
-      width="16"
-      height="16"
-      viewBox="0 0 16 16"
-      fill="none"
-    />
-  </span>
-)
-
 export const LiveIconComponentStory: StoryObj = {
```

Comment:
Boyscouting if you're up for it:

```suggestion
type Story = StoryObj<typeof meta>

// Update subsequent stories too
export const LiveIconComponentStory: Story = {
```

This ensures type safety where the consuming stories can inherit the props from the meta :)